feat(DATAARTS): add resource dataarts architecture model (#3836)
huaweicloud/terraform-provider-huaweicloud: docs/resources/dataarts_architecture_model.md

  workspace_id = var.workspace_id
  name         = var.name
  type         = "THIRD_NF"
  physical     = true
  dw_type      = "DWS"
}

resource "huaweicloud_dataarts_architecture_model" "logic_model"{
  workspace_id = var.workspace_id
  name         = var.name
  type         = "THIRD_NF"
  physical     = false
}
```

## Argument Reference

The following arguments are supported:

* `region` - (Optional, String, ForceNew) Specifies the region in which to manage the model.
  If omitted, the provider-level region will be used. Changing this parameter will create a new model.

* `workspace_id` - (Required, String, ForceNew) Specifies the workspace ID which the model in.
  Changing this parameter will create a new model.

* `physical` - (Required, Bool, ForceNew) Specifies the model is physical or logical.
  When the value is **true**, it means physical model.
  Changing this parameter will create a new model.

* `name` - (Required, String) Specifies the model name.

* `type` - (Required, String) Specifies the model type. The valid values are **THIRD_NF** and **DIMENSION**.

* `description` - (Optional, String) Specifies the description of model.

> * `dw_type` - (Optional, String) Specifies the data connection type. This parameter should specify when

Reviewer: ```suggestion
* `dw_type` - (Optional, String) Specifies the data connection type. This parameter is mandatory when
```
